package-lock.json

```diff
@@ -1,17939 +1,8 @@
 {
   "name": "webpack-dev-server",
   "version": "4.1.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
We use `"lockfileVersion": 2`, it would be nice if you can commit package-lock.json with npm@7, to avoid large diffs.

Reply:
Of course, my bad. Done.